src/view/ChewingEditor.cpp

```diff
@@ -223,8 +230,12 @@ void ChewingEditor::setupImport()
         model_, SIGNAL(importCompleted(bool, const QString&, size_t, size_t)),
         ui_.get()->notification, SLOT(notifyImportCompleted(bool, const QString&, size_t, size_t))
     );
+
+    // initialize import_count
+    import_count = 0;
```

Comment:
Can you use "static" qualifier to mark "import_count" as a static variable? So, it would be accumulated every time `ChewingEditor::finishFileSelection` is invoked.